birdhouse/default.env

```diff
@@ -31,17 +31,76 @@ export DELAYED_EVAL="
   PAVICS_FQDN_PUBLIC
   DATA_PERSIST_SHARED_ROOT
   WPS_OUTPUTS_DIR
+  SERVER_NAME
+  SERVER_DESCRIPTION
+  SERVER_INSTITUTION
+  SERVER_SUBJECT
+  SERVER_TAGS
+  SERVER_DOCUMENTATION_URL
+  SERVER_RELEASE_NOTES_URL
+  SERVER_SUPPORT_URL
+  SERVER_LICENSE_URL
 "
 
-
-export SERVER_NAME=PAVICS
-export SERVER_DESCRIPTION="
+# Server Identification Details
+#   Following definitions should definitenly be updated.
+#   Previous defaults are defined for backward-compatibility.
+#   If not overridden explicitly by their non '__' prefixed variant,
+#   a WARN message will be displayed by pavics-compose.
+export __DEFAULT__SERVER_NAME="PAVICS"
```

Comment:
I think all the new `__DEFAULT__` vars do not have to be exported because they should not be used in any sub-shells (scripts).  They are directly used here in `default.env` and `env.local` only.